conf/crontab.ugly

```diff
@@ -77,5 +80,8 @@ MAILTO=cron-!!(*= $site *)!!@mysociety.org
 13 2 * * * !!(*= $user *)!! run_management_command core_export_to_popolo_json /data/vhost/!!(*= $vhost *)!!/media_root/popolo_json/ http://www.pa.org.za
 30 3 * * * !!(*= $user *)!! run_management_command core_export_to_popolo_json --pombola /data/vhost/!!(*= $vhost *)!!/media_root/popolo_json/ http://www.pa.org.za
 
+# Sync EveryPolitician UUIDs to local DB
+30 10 * * * !!(*= $user *)!! output-on-error run_management_command core_sync_everypolitician_uuid
```

Comment:
I think the management command has a different name now - shouldn't it be: `south_africa_sync_everypolitician_uuid`?

Reply:
Oh yes, of course! Good catch! 😟